packages/workbox-background-sync/src/Queue.ts

```diff
@@ -211,7 +235,8 @@ class Queue {
       entry.metadata = metadata;
     }
 
-    await this._queueStore[`${operation}Entry`](entry);
+    await this._queueStore[
+        `${operation}Entry` as 'pushEntry' | 'unshiftEntry'](entry);
```

Comment:
I don't think I've seen this syntax before—this is how you tell TypeScript that `operation` might be either `push` or `unshift`?

Reply:
The method name called was being dynamically generated as a way to reduce total code size. I could add an `if` or `switch` statement (which I'm fine with as it's not *that* much more code), but specifying the possible method names seemed to make Typescript happy, so I opted to do that rather than refactoring -- at least for now.

Reply:
It's fine as-is.